### ISSUE TITLE
Add package for Raspbian buster

### ISSUE THREAD

**Issue by doe300** [doe300/VC4C#138]
Raspbian buster upgraded the preinstalled LLVM version from 3.9 to 7.0. More importantly, the version 3.9 is not available in the package repositories anymore.
Therefore, we need to upgrade the LLVM version the debian packages are built against.

There are 3 options:
- Leave existing package as-is and add new package for buster
- Upgrade existing package to a LLVM version which is provided in the standard repository for both stretch and buster, e.g. LLVM 6 (in buster main repository and in stretch-backports).
- Upgrade supported OS version to Raspbian buster, drop support for old versions. This would allow to completely move to latest LLVM (7.0) from the repository as well as move our code to C++17, since the default LLVM (7.0) and GCC (8.3) fully support it.

See https://github.com/doe300/VC4CL/issues/17

@nomaddo , any thoughts on this? Would it be possible to update the Docker images?

**Comment by doe300** [comment]
I did some (quick) tests, but alas upgrading the clang/llvm version is not that easy.

clang/llvm-6.0 is available for both stretch (via backports) and buster and is therefore a good candidate.
I also managed to modify the CI docker image to successfully build with clang/llvm-6.0.
But the resulting `vc4c` package does not work on stretch, since it requires newer glibc/libstdc++ (symbol) versions.

I will try to fix this in the next days and if this does not work, upgrade my local Raspberry Pis, drop support for stretch and upgrade to the "main" clang/llvm version for buster (currently 7.0).
